Frontend/src/app/components/event-list/event-list.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
+import { Event } from "../event";
+import { EventService } from "../../services/event.service";
+import { MatPaginator, MatTableDataSource, MatSort } from "@angular/material";
+import {EVENTS} from "../mock-events";
+import {Router} from "@angular/router";
+
+@Component({
+  selector: 'app-event-list',
+  templateUrl: './event-list.component.html',
+  styleUrls: ['./event-list.component.css']
+})
+export class EventListComponent implements OnInit {
+  events: Event[];
+  displayedColumns: string[] = ['name', 'teamSize', 'checkpointCount', 'created'];
+  dataSource = new MatTableDataSource<Event>(EVENTS);
```

Comment:
Look, jeigu jau susikurei event service tai panaudok cia, o ne tiesiai uzmockintus eventus imk i data source, removink ta EVENTS importa nes nereiks ir dataSource sukurk ngOnInit po to kai is service gausi events. 

Jeigu iliustruot su real life example, tai tu nusitiesei kelia, bet vistiek vaziuoji per pievas 😄